test/OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests/OtlpMetricsExporterTests.cs

```diff
@@ -986,30 +1006,30 @@ void AssertExemplars<T>(T value, Metric metric)
 
             if (typeof(T) == typeof(long))
             {
-                Assert.Equal((long)(object)value, exemplar.LongValue);
+                Assert.Equal((long)(object)value, otlpExemplar.AsInt);
```

Comment:
Note that this line and below, the previous code was validating against the In-Memory-Exporter exemplar instead of the otlp serialized exemplar.